awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsAppSignalsCustomizerProvider.java

```diff
@@ -57,6 +58,7 @@ public class AwsAppSignalsCustomizerProvider implements AutoConfigurationCustomi
       Logger.getLogger(AwsAppSignalsCustomizerProvider.class.getName());
 
   public void customize(AutoConfigurationCustomizer autoConfiguration) {
+    grpcProtocolByDefault(autoConfiguration);
```

Comment:
Not blocking: Consider rename method to `customizeConfigurationPropertiesSupplier` in line with the other methods, and allowing us to expand this method in future, if we need to add other properties.